fontir/src/ir.rs

```diff
@@ -95,6 +98,37 @@ pub struct MiscMetadata {
     pub created: Option<DateTime<Utc>>,
 }
 
+/// IR representation of kerning.
+///
+/// In UFO terms, roughly [groups.plist](https://unifiedfontobject.org/versions/ufo3/groups.plist/)
+/// and [kerning.plist](https://unifiedfontobject.org/versions/ufo3/kerning.plist/) combined.
+#[derive(Serialize, Deserialize, Debug, Default, Clone, PartialEq, Eq)]
+#[serde(from = "KerningSerdeRepr", into = "KerningSerdeRepr")]
+pub struct Kerning {
+    pub groups: HashMap<GroupName, HashSet<GlyphName>>,
```

Comment:
since these are only the groups that have to do with kerning, we don't actually need to keep the "public.kern1." and "public.kern2." prefixes in the GroupName, there's no risk of clashing with non-kerning groups

Reply:
Nice to be able to directly lookup what you get from kerning.plist so I'm inclined to retain. Please reopen if you disagree.